extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.NetStandard.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+	<TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
+	<AssemblyName>AWSSDK.Extensions.CloudFront.Signers</AssemblyName>
+	<PackageId>AWSSDK.Extensions.CloudFront.Signers</PackageId>
+	<Version>4.0.1.0-preview</Version>
+	<Title>AWSSDK.Extensions.Cloudfront.Signers</Title>
+	 <DefineConstants>$(DefineConstants);NETSTANDARD;AWS_ASYNC_API</DefineConstants>
+	<GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+	<GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+	<GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+	<GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+	<GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+	<GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+	<GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+	<GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+  <ItemGroup>
```

Comment:
Project file should be marked as trimmable for the .NET 8 target.
```
  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
    <IsTrimmable>true</IsTrimmable>
  </PropertyGroup>
```

Reply:
good call, added.